registrations/migrations/0012_mandatory_fields_model.py

```diff
@@ -0,0 +1,117 @@
+# Generated by Django 3.2.18 on 2023-04-21 05:31
+
+from django.db import migrations, models
+
+from registrations.models import MandatoryField
+
+
+def create_initial_mandatory_fields(apps, schema_editor):
+    MandatoryField.objects.create(
```

Comment:
In migrations you shouldn't import the model directly, but use a historical version instead, because later on the MandatoryFields model might be different than what is expected by the state of the DB in this moment.

So:

```python
MandatoryField = apps.get_model("registration", "MandatoryField")
MandatoryField.objects.create(...)
...
```